pilot/cmd/pilot-agent/main.go

```diff
@@ -150,7 +150,7 @@ var (
 	skipParseTokenEnv = env.RegisterBoolVar("SKIP_PARSE_TOKEN", false,
 		"Skip Parse token to inspect information like expiration time in proxy. This may be possible "+
 			"for example in vm we don't use token to rotate cert.").Get()
-	proxyXDSViaAgent = env.RegisterStringVar("ISTIO_META_PROXY_XDS_VIA_AGENT", "",
+	proxyXDSViaAgent = env.RegisterStringVar("ISTIO_META_PROXY_XDS_VIA_AGENT", "true",
```

Comment:
Should be more cautious about this, it can be severe if there is any issue.

Reply:
Can we add `ISTIO_META_PROXY_XDS_VIA_AGENT` to those two profiles for now? I think we still need